PoseStamped hand_domino
PoseStamped target_location
String left_right
---


